charts/common/aws-authservice/cognito/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
afaik, this auth service is not specific to cognito. The envoyfilter and logout service can be used by any oidc auth setup. Create the chart directly under authservice directory

Reply:
https://github.com/awslabs/kubeflow-manifests/pull/356/files#diff-57bed61a97b7973a2c9e8e86d1254e3699d5fa22dbdb951fabaaa2bcc95a36c9R27

Looks like this part could be specific to cognito?

```
kind: Deployment
metadata:
  name: aws-authservice
   ...
    spec:
      ...
        image: public.ecr.aws/c9e4w0g3/cognito/aws-authservice:v1.0.0
```

Reply:
Good catch. Looks like the image was published this way because it was used in resolving #62 Cognito logout bug also this is the only service we have tested with but #305 and #353  confirms this service can be used with ALB+OIDC auth in general